modules/ROOT/pages/deriving-container.adoc

```diff
@@ -0,0 +1,94 @@
+= Deriving from Fedora CoreOS as a bootable container image
+
+== Bootable containers overview
+
+The operating system images used for Fedora CoreOS are available as a standard container image; there is one per stream:
+
+- quay.io/fedora/fedora-coreos:stable
+- quay.io/fedora/fedora-coreos:next
+
+etc.
+
+== Creating custom builds
+
+See the https://github.com/coreos/layering-examples[layering examples] git repository.  While the examples there tend to use Container/Dockerfile style builds, you can use any tool which can build containers and push the result to a registry.
+
+== Using Ignition to retarget an instance
```

Comment:
```suggestion
== Using Ignition to rebase an instance
```

Maybe rebase instead?

Reply:
While I love the term "rebase" myself...it feels obscure because it's a thing not present in docker/podman/k8s.